python/publish_transforms_to_s3.py

```diff
@@ -0,0 +1,38 @@
+"""
+Python Script to Publish all Transforms (for every DW) to S3
+"""
+import json
+import rasgotransforms
+import boto3
+
+
+def to_camel_case(snake_str):
+    components = snake_str.split('_')
+    return components[0] + ''.join(x.title() for x in components[1:])
+
+
+def get_some_transforms(dw_type: str):
+    return [
+        {**x.__dict__, **{'dw_type': dw_type}}
```

Comment:
```suggestion
        {**x.__dict__, 'dw_type': dw_type}
```

Why all the explodin'?